btsieve/src/route_factory.rs

```diff
@@ -108,6 +108,17 @@ where
     let query_repository = warp::any().map(move || Arc::clone(&query_repository));
     let query_result_repository = warp::any().map(move || Arc::clone(&query_result_repository));
 
+    // validate query id function
+    let validate_query_id = warp::path::param::<String>().and_then(|id: String| {
+        if id.len() > 40 {
```

Comment:
Can we have a `const` declared instead of a magic number please.  A comment stating why 40 was chosen on the const would make me smile.

Reply:
Good point! It should be clear now :smile: